lib/reporting/command_line_options.rb

```diff
@@ -27,12 +32,23 @@ def parse!(argv, out: STDOUT)
         end
 
         opts.on('--week=DATE', <<~STR.squish) do |date_v|
-          run the report for the week (Sun-Sat) that includes the date in YYYY-MM-DD format'
+          run the report for the week (Sun-Sat) that includes the date in YYYY-MM-DD format
         STR
           date = Date.parse(date_v)
           period = :week
         end
 
+        opts.on('--month=DATE', <<~STR.squish) do |date_v|
+          run the report for the month that includes the date in YYYY-MM-DD format.
+          changes defaults to `--threads 10` and `--slice 1 hour` to reduce likelihood of timeout
+        STR
+          date = Date.parse(date_v)
+          period = :month
+
+          slice = 1.hour if slice == 3.hours
+          threads = 10 if threads == 5
```

Comment:
related to https://github.com/18F/identity-idp/pull/8318/files#r1183051547, I think we could make this clearer if we ```suggestion
          slice ||= 1.hour
          threads ||= 10
```